Sources/Answer.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Answer.swift
+//  SwiftStack
+//
+//  Created by FelixSFD on 09.12.16.
+//
+//
+
+import Foundation
+
+public class Answer: Post {
+    
+    // - Initializers
+    
+    public required init(dictionary: [String : Any]) {
+        super.init(dictionary: dictionary)
+        
+        self.accepted = dictionary["accepted"] as? Bool
+        self.answer_id = dictionary["answer_id"] as? Int
+        self.awarded_bounty_amount = dictionary["awarded_bounty_amount"] as? Int
+        
+        if let users = dictionary["awarded_bounty_users"] as? [[String: Any]] {
+            var array = [User]()
+            
+            for user in users {
+                let tmpUser = User(dictionary: user)
+                array.append(tmpUser)
```

Comment:
I think it would be easier to read without it, but it doesn't really matter I guess.